tests/conftest.py

```diff
@@ -165,6 +179,17 @@ def release(
     zfs: libzfs.ZFS
 ) -> 'libioc.Release.ReleaseGenerator':
     """Return the test release matching the host release version."""
-    return libioc.Release.Release(
-        name=host.release_version, host=host, logger=logger, zfs=zfs
+    if host.release_version.endswith("RELEASE"):
+        target_release = host.release_version
+    else:
+        target_release = "12.0-RELEASE"
+
+    release = libioc.Release.Release(
+        name=target_release, host=host, logger=logger, zfs=zfs
     )
+    return release
+
+
+import release_mirror_cache
+release_mirror_cache.run_thread(8081)
```

Comment:
how do we opt out of this?

Reply:
Each test can decide whether to use the mocked Distribution module from the fixture or the imported one.